docs/ANNOTATION.md

```diff
@@ -0,0 +1,38 @@
+# Annotation
+
+## Tools
+
+Within Sarek, annotation is done using snpEff, VEP, or both consecutively:
+- `--tools snpEff`
+  - To annotate using `snpEff`
+- `--tools VEP`
+  - To annotate using `VEP`
+- `--tools snpEff,VEP`
+  - To annotate using `snpEff` and `VEP`
+- `--tools merge`
+  - To annotate using `snpEff` then `VEP`
+
+## Using containers
+
+Sarek has already designed containers with `snpEff` and `VEP` files for `GRCh37` and `GRCh38`.
+Default settings will run using these containers.
+
+## Using cache
+
+Both `snpEff` and `VEP` enable usage of cache.
+If cache is available on the machine where Sarek is run, it is possible to run annotation using cache.
+You can specify the cache directory using `--snpEff_cache` and `--vep_cache` in the command lines or within configuration files.
+But the cache will only be used whenn `--annotation_cache` is specified (either in command lines or in a configuration file).
+
+Example:
+```
+nextflow run annotate.nf --tools snpEff --annotateVCF file.vcf.gz --genome GRCh38 --snpEff_cache /Path/To/snpEffCache --annotation_cache
+nextflow run annotate.nf --tools VEP --annotateVCF file.vcf.gz --genome GRCh38 --vep_cache /Path/To/vepCache --annotation_cache
+```
+
+## Downloading cache
+
+An helper script has been designed to help downloading snpEff and VEP cache.
```

Comment:
You could add here as well that this is only necessary if one is not running on uppmax and do not wish to use the specific containers.